app/src/main/java/org/mozilla/fenix/search/toolbar/SearchSelectorMenu.kt

```diff
@@ -65,3 +66,16 @@ class SearchSelectorMenu(
         )
     }
 }
+
+/**
+ * Interface for search selector action.
+ */
+interface SearchSelectorInteractor {
```

Comment:
Let's move this to its own file

Reply:
Could you explain a bit more? I see a lot of classes (that relate to the UI, like views and delegates) that take interactor as a parameter declaring the interface inside the file. Well, even this one was chipped away from `ToolbarInteractor`, which is the same file with `ToolbarView`.

Reply:
This is more about moving the Interactor interface into its own file and keeping them out of the views. We have found that there will be times where the Interactor interface could be reused in a variety of places so it would make more sense to keep them to their own file from the get go.

I would imagine you are referring to Bookmark and History views for what you're seeing, but those are also some of our oldest views that we don't actively polish compare to the Home screen. Other than that I would say we would rather keep files smaller for better development ergonomics if there's a combination of classes, functions and interfaces that are all in one place. I could also imagine the views are more volatile to changes and deletion compare to its own separate file that holds the interactor interface.

Ultimately, this is just about developer ergonomics. It's easier to work with smaller files and you aren't at risk of refactoring in the future.